### ISSUE TITLE
Default status of a new issue should be unconfirmed?

### ISSUE THREAD

**Issue by wilzbach** [CyberShadow/bugzilla-meta#11]

**Comment by CyberShadow** [comment]
Bug triaging could be a worthwhile idea!

We could have an initial bug state that is transitioned to a different one once someone else reads the bug report and makes sure it's complete and reproducible.

**Comment by wilzbach** [comment]
I just saw that Mozilla even goes one step further: https://wiki.mozilla.org/BMO/UserGuide#How_to_apply_for_upgraded_permissions

> If you want canconfirm, you can add it yourself using triage request form.
> If you want editbugs, file a new bug, including:

If have never seen any/much abuse on issues.dlang.org and I think that we can still allow both permissions by default. Imho it's actually one of the advantages over GH and I doubt that someone is going to self-confirm his bug (and if he does - he's probably one of the dmd/druntime/phobos devs) that just created the bug as personal tracking.

**Comment by CyberShadow** [comment]
Yep, I didn't mean to also introduce permission levels (other than admin). Most users file bugs, people who consider themselves to know what they're doing can triage/confirm.
